src/Aspire.Dashboard/Components/ResourcesGridColumns/SourceColumnDisplay.razor

```diff
@@ -3,48 +3,27 @@
 @using Aspire.Dashboard.Resources
 @inject IStringLocalizer<Columns> Loc
 
-@if (Resource.IsProject() && Resource.TryGetProjectPath(out var projectPath))
+@if (ContentAfterValue is not null)
```

Comment:
Is it possible to put this `@if` inside the `<GridValue>`, around the `<ContentAfterValue>`?